app/helpers/compute_resources_vms_helper.rb

```diff
@@ -142,8 +142,12 @@ def show_vm_name?
     controller_name != 'hosts' && controller_name != 'compute_attributes'
   end
 
-  def new_host?(host)
-    host.try(:new_record?)
+  def new_host?(host, options = {})
+    if host.nil? && options[:nil_is_new]
```

Comment:
Do we need the `:nil_is_new` option or can we just return true for any `new_host?(nil)`? `nil_is_new` is really hard to read in my opinion.